world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+(:name find-android
+ :title "Find the Android"
+ :description "Locate the android rumoured to live in the mountain caves"
+ :invariant T
+ :condition quest:all-complete
+ :on-activate (fi-check pot-check tent-check sign-main-check sign-check landslide-check fire-check beacon-check corpse-check cave-in-check android-check home-check home-tent-check))
+(quest:interaction :name fi-check :interactable fi :dialogue "
+~ fi
+| (:annoyed)Next time I'll bring my own climbing gear.
+~ player
+| Suit yourself. But this was your idea.
+~ fi
+| Tracking the android was my idea. Talking to Harris was my idea. Clinging to your back while you scale a vertical cliff face - (:shocked)definitely not my idea.
+~ player
+| Well, we're here now. Can I proceed?
+~ fi
+| You know you don't need to ask me that, of all people? But yes, you are permitted.
+| Though I'm staying right here. Let me know what you find. And be careful.")
+(quest:interaction :name pot-check :interactable pot :dialogue "
+~ player
+| (The cooking pot infers human habitation.
+| (And this is a great spot for a lookout...)
+| ! eval (activate 'fi-tent)
+| ! eval (complete 'fi-check)
+")
+; | ! eval (let (fi-tent-talk true)) - works to set
+(quest:interaction :name tent-check :interactable tent :dialogue "
+~ player
+| (There's no one inside, but the bedroll suggests it was recently occupied.)
+| (Does an android require a bedroll? I recharge standing up.)
```

Comment:
I like the suggested edit 👍 
"People/persons" sounds good

Unsure at the literal start of the game whether the Stranger would distance herself from people - would need more thought in consideration of the plot and character development. However, for this demo I think it could be an interesting hook right from the start to set the scene of the android being in service to the people to a degree.

This quest might not be in the final game, but if so it feels like something mid-game, which I don't think is a bad thing to drop the demo player into the middle of - makes it feel like stuff has already happened and they're straight into a cool mission (as long as there is enough orientation/tutorialisation still)

Reply:
Sounds good.